project-base/storefront/pages/wishlist.tsx

```diff
@@ -35,7 +35,12 @@ const WishlistPage: NextPage<ServerSidePropsType> = () => {
 export const getServerSideProps = getServerSidePropsWrapper(
     ({ redisClient, domainConfig, t }) =>
         async (context) =>
-            initServerSideProps({ context, redisClient, domainConfig, t }),
+            initServerSideProps({
+                context,
+                redisClient,
+                domainConfig,
+                t,
+            }),
```

Comment:
This seems like an unnecessary change. You can resolve this thread by yourself, but I just wanted to write it to let you know that such changes should be avoided, as they do not bring anything beneficial.